server/src/main/java/io/seata/server/store/StoreConfig.java

```diff
@@ -29,13 +29,19 @@
     private static final Configuration CONFIGURATION = ConfigurationFactory.getInstance();
 
 
-    // default 16kb
+    /**
+     * default 16kb
+     */
```

Comment:
Please use capital for the first letter, and use dot as end. And description should be more clearly.